src/components/BookmarkContainer.jsx

```diff
@@ -0,0 +1,99 @@
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable react/jsx-indent */
+/* eslint-disable react/jsx-closing-tag-location */
+/* eslint-disable react/no-array-index-key */
+/* eslint-disable react/jsx-wrap-multilines */
```

Comment:
Do you think we can fix these issues rather than disabling the eslint warnings?

Reply:
fixed @valenstical